components/feature/prompts/src/main/java/mozilla/components/feature/prompts/file/FilePicker.kt

```diff
@@ -113,8 +113,9 @@ internal class FilePicker(
      * @param requestCode The code of the app that requested the intent.
      * @param intent The result of the request.
      */
-    fun onActivityResult(requestCode: Int, resultCode: Int, intent: Intent?) {
-        val request = getActivePromptRequest() ?: return
+    fun onActivityResult(requestCode: Int, resultCode: Int, intent: Intent?): Boolean {
+        var resultHandled = false
```

Comment:
@Amejia481 you may be interested in reviewing this change. 🙂

Reply:
Sure, I will take a look today :)

Reply:
Looks good ✅ !

Reply:
Thanks!